### PR TITLE
Pretty docs

### PR DESCRIPTION
**Pull request by anonymoussprocket**
This will absorb and replace /doc

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+...
+```
+
+#### Extended metadata
+
+As mentioned in the [Datasource configuration](#datasource-configuration) section, it's possible to provide additional details for the items Conseil manages. 
+
+### Tezos Chain Data Query
+
+Data requests in Conseil v2 API are sent via `POST` command and have a prefix of `/v2/data/<platform>/<network>/<entity>`. The items in curly braces match up to the `name` property of the related metadata results. There additional header to be set in the request: `Content-Type: application/json`.
```
Comment (ivanopagano) [inline docs/README.md]: >The items in _curly braces_ match up...

actually _angle brackets_ ?

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+The query grammar allows for some fine-grained data extraction.
+
+#### `fields`
+
+It is possible to narrow down the fields returned in the result set by listing just the ones that are necessary. The `fields` property of the query is an array of `string`, these values are the `name`s from the `/v2/metadata/<platform>/<network>/<entity>/attributes/` metadata response.
+
+#### `predicates`
+
+`predicates` is an array containing objects. The inner object as several properties:
```
Comment (ivanopagano) [inline docs/README.md]: >The inner object _has_ several properties:

typo

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+- `field` – attribute name from the metadata response.
+- `operation` – one of: in, between, like, lt, gt, eq, startsWith, endsWith, before, after.
+- `set` – an array of values to compare against. `in` requires two or more elements, `between` must have exactly two, the rest of the operations require a single element.
+- `inverse` – boolean, setting it to `true` applied a `NOT` to the operator
```
Comment (ivanopagano) [inline docs/README.md]: I'm starting to wonder (late for doing so) if `negate` would better pass the meaning across?

On the other hand a boolean value named "negate" might become confusing

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+- `precision` – for numeric field matches this specifies the number of decimal places to round to.
+
+#### `limit`
+
+Specifies the maximum number of records to return.
+
+#### `orderBy`
+
+Sort condition, multiple may be supplied for a single query. Inner obejct(s) will contain `field` and `direction` properties. The former is `name` from the `/v2/metadata/<platform>/<network>/<entity>/attributes/` metadata response, the latter is one of 'asc', 'desc'.
```
Comment (ivanopagano) [inline docs/README.md]: I see an small inconsistency in how keywords are emphasised like the predicate operation enumerations, which are plain text, the same used as an example for the predicates' set field, and here with the ordering values in single quotes.
Using a single consistent notation might help the reader

Comment (ivanopagano) [reply docs/README.md]: this applies to the aggregation's functions

File: docs/README.md (hunk 3 above)
Comment (vishakh) [reply docs/README.md]: Interesting point. Perhaps `invert` might also work?

**Pull request merged**